LobbyCompatibility/Patches/MenuManagerPostfix.cs

```diff
@@ -30,10 +30,7 @@ private static void Postfix(MenuManager __instance)
             return;
 
         LobbyCompatibilityPlugin.Logger?.LogInfo("Initializing menu UI.");
-
-        // Set challenge moon leaderboard title text to not wrap halfway through
-        __instance.leaderboardHeaderText.rectTransform.offsetMax = new Vector2(2000, __instance.leaderboardHeaderText.rectTransform.offsetMax.y);
```

Comment:
What is this change for? I don't see it mentioned in your PR.

Reply:
Moving it back to a postfix since only part of the code needed moving to the prefix